WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyRingViewModel.cs

```diff
@@ -113,6 +114,16 @@ private void CreateSegments(IEnumerable<Pocket> pockets, IExtendedList<PrivacyRi
 
 		PreviewItems.RemoveRange(1, PreviewItems.Count - 1);
 		PreviewItems.AddRange(list);
+
+		References.Clear();
+
+		var references =
+			list.GroupBy(x => (x.IsPrivate, x.IsNonPrivate, x.IsSemiPrivate, x.Unconfirmed))
```

Comment:
Shouldn't `IsSemiPrivate` come before `IsNonPrivate` here?

Reply:
@yahiheb fixed in latest commit.

BTW this doesn't affect functionality, since it's a tuple used as a group key, order of elements inside the tuple aren't relevant. But yeah for the sake of consistency I've changed it.